x-pack/test/functional/apps/maps/feature_controls/maps_spaces.ts

```diff
@@ -78,20 +78,13 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await spacesService.delete('custom_space');
       });
 
-      it(`returns a 404`, async () => {
+      it(`renders a not found message`, async () => {
```

Comment:
This test is currently skipped. Have you run it locally to ensure your changes work?

Reply:
Yes, ran locally to verify